scripts/cm_test_all_sandia

```diff
@@ -488,7 +489,8 @@ elif [ "$MACHINE" = "sogpu" ]; then
   INTEL_BASE_MODULE_LIST="sems-archive-env,sems-archive-cmake/3.17.1,sems-archive-gcc/7.2.0,sems-archive-<COMPILER_NAME>/<COMPILER_VERSION>"
   CLANG_BASE_MODULE_LIST="sems-archive-env,sems-archive-cmake/3.17.1,sems-archive-gcc/9.2.0,sems-archive-<COMPILER_NAME>/<COMPILER_VERSION>"
   CUDA_MODULE_LIST="sems-archive-env,sems-archive-cmake/3.17.1,sems-archive-gcc/7.2.0,sems-archive-<COMPILER_NAME>/<COMPILER_VERSION>"
-  CUDA11_MODULE_LIST="sems-cmake/3.23.1,sems-gcc/8.3.0,sems-<COMPILER_NAME>/<COMPILER_VERSION>"
```

Comment:
It is my understand this this takes care of the ordering issue already via:
`module load sems-cmake/3.23.1 sems-gcc/8.3.0 sems-cuda/11.1.0`

Reply:
I would have thought so as well, not sure why there are issue :(
When I experimented with manually loading the modules, I found that if I loaded sems-gcc/8.3.0 first then sems-cuda/11.1.0 it was successful; I didn't try loading cmake first. It could also be due to sourcing the archive version of the sems script prior.